packages/skills/carpark_detection/strategy.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""This module contains the strategy class."""
+import os
+from typing import Any, Dict, List, Tuple, TYPE_CHECKING
+
+from aea.protocols.oef.models import Description, Query
+from aea.skills.base import SharedClass
+
+if TYPE_CHECKING:
+    from packages.skills.carpark_detection.detection_database import DetectionDatabase
+    from packages.skills.carpark_detection.carpark_detection_data_model import CarParkDataModel
+
+else:
+    from carpark_detection_skill.detection_database import DetectionDatabase
+    from carpark_detection_skill.carpark_detection_data_model import CarParkDataModel
+
+DEFAULT_PRICE_PER_ROW = 2
+DEFAULT_CURRENCY_PBK = 'FET'
+DATE_ONE = "3/10/2019"
+DATE_TWO = "15/10/2019"
+
+
+class Strategy(SharedClass):
+    """This class defines a strategy for the agent."""
+
+    def __init__(self, **kwargs) -> None:
+        """
+        Initialize the strategy of the agent.
+
+        :param register_as: determines whether the agent registers as seller, buyer or both
+        :param search_for: determines whether the agent searches for sellers, buyers or both
+
+        :return: None
+        """
+        self.price_per_row = kwargs.pop('price_per_row') if 'price_per_row' in kwargs.keys() else DEFAULT_PRICE_PER_ROW
+        self.currency = kwargs.pop('currency_pbk') if 'currency_pbk' in kwargs.keys() else DEFAULT_CURRENCY_PBK
+        super().__init__(**kwargs)
+
+        self.db = DetectionDatabase(os.path.dirname(__file__))
+        self.data_price_fet = 2000
```

Comment:
Can you make it so these can be fed via the config and some default values are provided as constants if nothing is fed from configs (like I did for self.price_per_row)?